apps/win32/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.126.35",
+    "@warren-ms/react-native-icons": "^0.0.11",
```

Comment:
This should probably be in the main tester package's package.json. This one is only for the win32 tester.

Reply:
Also curious if this will be replaced with fluentui-system-icons later?

Reply:
After some more clean-up of the publishing code I'll be trying to get a PR into the official fluentui repo and probably publish this package as @fluentui/react-native-svg-icons. Their current web react is @fluentui/svg-icons.